packages/astroport-governance/src/generator_controller.rs

```diff
@@ -5,48 +5,58 @@ use serde::{Deserialize, Serialize};
 /// This structure describes the basic settings for creating a contract.
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
 pub struct InstantiateMsg {
-    /// contract owner
+    /// Contract owner
     pub owner: String,
-    /// the vxASTRO token contract address
+    /// The vxASTRO token contract address
     pub escrow_addr: String,
-    /// generator contract address
+    /// Generator contract address
     pub generator_addr: String,
-    /// factory contract address
+    /// Factory contract address
     pub factory_addr: String,
-    /// max number of pools that can receive an ASTRO allocation
+    /// Max number of pools that can receive ASTRO emissions at the same time
     pub pools_limit: u64,
 }
 
-/// This structure describes the execute messages of the contract.
+/// This structure describes the execute messages available in the contract.
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
 #[serde(rename_all = "snake_case")]
 pub enum ExecuteMsg {
+    /// Vote allows a vxASTRO holder to cast votes on which generators should get ASTRO emissions in the next epoch
     Vote {
         votes: Vec<(String, u16)>,
     },
+    /// TunePools transforms the latest vote distribution into alloc_points which are then applied to ASTRO generators
     GaugePools {},
+    /// ChangePoolsLimit changes the max amount of pools that can be voted at once to receive ASTRO emissions
     ChangePoolsLimit {
         limit: u64,
     },
-    /// Propose a new owner for the contract
+    /// ProposeNewOwner proposes a new owner for the contract
     ProposeNewOwner {
+        /// Newly proposed contract owner
         new_owner: String,
+        /// The timestamp when the contract ownership change expires
         expires_in: u64,
     },
-    /// Remove the ownership transfer proposal
+    /// DropOwnershipProposal removes the latest contract ownership transfer proposal
     DropOwnershipProposal {},
-    /// Claim contract ownership
+    /// ClaimOwnership allows the newly proposed owner to claim contract ownership
     ClaimOwnership {},
 }
 
-/// This structure describes the query messages of the contract.
+/// This structure describes the query messages available in the contract.
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
 #[serde(rename_all = "snake_case")]
 pub enum QueryMsg {
+    /// UserInfo returns information about a voter and the generators they voted for
     UserInfo { user: String },
+    /// TuneInfo returns information about the latest generators that were voted to receive ASTRO emissions
     GaugeInfo {},
+    /// Config returns the contract configuration
     Config {},
+    /// PoolInfo returns the latest voting power allocated to a specific pool (generator)
     PoolInfo { pool_addr: String },
+    /// PoolInfo returns the voting power allocated to a specific pool (generator) at a specific timestamp
```

Comment:
Here we fetch pool information at the period number not at the specific timestamp